docs/database/database-spans.md

```diff
@@ -478,6 +480,32 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Context propagation
+
+**Status**: [Development][DocumentStatus]
+
+### SQL commenter
+
+Instrumentations MAY propagate context using [SQL commenter](https://google.github.io/sqlcommenter/spec/) by injecting comments into SQL queries before execution. Context propagation SHOULD NOT be enabled by default, but instrumentation MAY allow users to opt into it.
+
+The instrumentation implementation MAY choose to either **append** the comment to the end of the query or **prepend** the comment at the beginning of the query, depending on the specific database system's requirements or preferences.
+
+The instrumentation SHOULD allow users to pass a list of propagators to overwrite the default propagators.
+
+The instrumentation MAY provide the following propagators that users can use to overwrite the default propagators:
```

Comment:
There is a chance that I missed something, but it is not obvious what are `default propagators` here.

Reply:
I guess the global propagator as usual